packages/functional-tests/pages/login.ts

```diff
@@ -219,6 +221,34 @@ export class LoginPage extends BaseLayout {
     return this.page.fill(selectors.PASSWORD, password);
   }
 
+  async showPasswordMouseAction() {
+    return Promise.all([
+      await this.page.locator(selectors.PASSWORD).type('password'),
```

Comment:
Not sure if it's intentional and doesn't break anything, but having this here means the password is typed in twice, even though I don't think it's needed the second time for the test added in this PR.  Moreover, there's a `setPassword` function right above this, why not use that instead?

Reply:
Added a line to clear the password after first `expect`. I do not want to use the setPassword() because this test is solely for the mousedown action to show `text` and `password` input field.

Reply:
> I do not want to use the setPassword() because this test is solely for the mousedown action to show text and password input field.

I don't understand.  But it was a nit anyway.